### PR TITLE
Normalize binary responses

### PR DESCRIPTION
**Pull request by cilim**

### CODE REVIEW

File: lib/dox/formatters/plain.rb [hunk 1]
```diff
@@ -2,7 +2,26 @@ module Dox
...
+        normalize(body)
+      end
+
+      private
+
+      def normalize(data)
+        case data
+        when String
+          to_utf8(data)
+        else
```
Comment (melcha) [inline lib/dox/formatters/plain.rb:14]: Can it be anything else but string?

File: lib/dox/formatters/plain.rb [hunk 2]
```diff
@@ -2,7 +2,26 @@ module Dox
...
+          data
+        end
+      end
+
+      def to_utf8(stream)
```
Comment (melcha) [inline lib/dox/formatters/plain.rb:19]: Add a test for this case.

Comment (cilim) [reply lib/dox/formatters/plain.rb:19]: there are no fine-grained specs at all at the moment. Should I write unit tests perhaps?

File: lib/dox/formatters/plain.rb (hunk 1 above)
Comment (cilim) [reply lib/dox/formatters/plain.rb:14]: I don't think it can actually...will remove it 👍

File: lib/dox/formatters/plain.rb (hunk 2 above)
Comment (melcha) [reply lib/dox/formatters/plain.rb:19]: Yes.

**Pull request merged**